### PR TITLE
[Implement] add flink Execution plan print

### PR DESCRIPTION
**Pull request by zhuangchong**
---

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] Change does not need document change, or I will submit document change to https://github.com/InterestingLab/seatunnel-docs later

### CODE REVIEW

File: seatunnel-core/seatunnel-core-base/src/main/java/io/github/interestinglab/waterdrop/config/ConfigBuilder.java [hunk 1]
```diff
@@ -154,23 +153,16 @@ public void checkConfig() {
     }
 
     public <T extends Plugin> List<T> createPlugins(PluginType type) {
-
```
Comment (leo65535) [inline seatunnel-core/seatunnel-core-base/src/main/java/io/github/interestinglab/waterdrop/config/ConfigBuilder.java:157]: Hi, these changes are not related to the patch purpose, it's better to keep it.

Comment (zhuangchong) [reply seatunnel-core/seatunnel-core-base/src/main/java/io/github/interestinglab/waterdrop/config/ConfigBuilder.java]: done.

**Comment by leo65535** [comment]
+1, LGTM

**Review by leo65535 — APPROVED** [review]

**Review by CalvinKirs — APPROVED** [review]
LGTM
